scripts/optional.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# This script will download and add domains from the rep to whitelist.txt file.
+# Project homepage: https://github.com/anudeepND/whitelist
+# Licence: https://github.com/anudeepND/whitelist/blob/master/LICENSE
+# Created by Anudeep
+#================================================================================
+TICK="[\e[32m ✔ \e[0m]"
+
+
+echo -e " \e[1m This script will download and add domains from the repo to whitelist.txt \e[0m"
+sleep 1
+echo -e "\n"
+
+if [ "$(id -u)" != "0" ] ; then
+	echo "This script requires root permissions. Please run this as root!"
+	exit 2
+fi
+
+if [ "$(dpkg-query -W -f='${Status}' gawk 2>/dev/null |  grep -c "ok installed")" -eq 0 ];
+then
+  echo -e " [...] \e[32m Installing gawk... \e[0m"
+  apt-get install gawk -qq > /dev/null
+  wait
+  echo -e " ${TICK} \e[32m Finished \e[0m"
+fi
+
+
+curl -sS https://raw.githubusercontent.com/anudeepND/whitelist/master/domains/optional-list.txt | sudo tee -a /etc/pihole/whitelist.txt >/dev/null
```

Comment:
There's no need for `sudo tee`, the script is already running as root:
```suggestion
curl -sS https://raw.githubusercontent.com/anudeepND/whitelist/master/domains/optional-list.txt >>/etc/pihole/whitelist.txt
```